js/src/hooks/useAutoCheckEnhancedConversionTOS.js

```diff
@@ -31,10 +31,7 @@ const useAutoCheckEnhancedConversionTOS = () => {
 
 	useWindowFocusCallbackIntervalEffect( checkStatus, 30 );
 
-	return {
-		startEnhancedConversionTOSPolling,
-		stopEnhancedConversionTOSPolling,
-	};
+	return polling;
```

Comment:
We're no longer using any return value from this hook, so this could be removed. Alternately, we could return an object containing the polling status and the return value of `acceptedCustomerDataTerms()` as a convenience. So components wouldn't need to use this alongside the `acceptedCustomerDataTerms()` hook, ex:

```suggestion
	return {
		isPolling,
		acceptedCustomerDataTerms,
		hasFinishedResolution
	};
```

Reply:
Sounds good! Updated @joemcgill